### ISSUE TITLE
Is the development completed?

### ISSUE THREAD

**Issue by zihaozhang9** [Ziems/ProxylessNAS#1]
Thank you very much for exposing the code. Your code works. May I ask if you haven't finished that part of the work? Recently I wanted to do the same. I can help you with the project.

**Comment by Ziems** [comment]
Yes there are definitely some major parts that I have not finished.  I have not tried it with many different layers or ran it for a long period of time. Of course you are free to help! Probably the next step is to add support for more operations. Right now I think it just allows for conv and nothing else.

**Comment by zihaozhang9** [comment]
yes.Such as maxpool3x3、avgpool3x3、identity

**Comment by Ziems** [comment]
Exactly! Also, the second half of the paper, the part that optimizes for certain devices, has not been implemented yet either.
